src/main/java/org/proffart/football/training/api/impl/PlayerApiImpl.java

```diff
@@ -0,0 +1,46 @@
+package org.proffart.football.training.api.impl;
+
+import org.proffart.football.training.api.PlayerApi;
+import org.springframework.stereotype.Component;
+
+import javax.ws.rs.*;
+
+/**
+ * Author Artak Mnatsakanyan
+ * Date 9/13/16
+ * Time 3:11 PM
+ */
+@Component
+@Path("/player")
+public class PlayerApiImpl implements PlayerApi {
+
+    @POST
+    @Override
+    public String createPlayer(String name) {
```

Comment:
Player also have other properties which must be received for example 

- birthday
- height
- weight
- startedTrainings
- previousExperience  

also you have to change name into fullName